chains/evm/contracts/onRamp/OnRamp.sol

```diff
@@ -51,19 +51,22 @@ contract OnRamp is IEVM2AnyOnRampClient, ITypeAndVersion, Ownable2StepMsgSender
     bytes offRamp
   );
   event FeeTokenWithdrawn(address indexed feeAggregator, address indexed feeToken, uint256 amount);
-  /// RMN depends on this event, if changing, please notify the RMN maintainers.
   event CCIPMessageSent(
     uint64 indexed destChainSelector,
     uint64 indexed sequenceNumber,
     bytes32 indexed messageId,
     bytes encodedMessage,
-    Receipt[] verifierReceipts,
-    Receipt executorReceipt,
-    bytes[] receiptBlobs
+    Receipt[] receipts,
+    bytes[] verifierBlobs
   );
 
+  struct CCIPMessageSentEventData {
```

Comment:
This was introduced to resolve the stack-too-deep and feels cleaner than re-stating variables.